model/src/main/kotlin/Issue.kt

```diff
@@ -77,9 +77,14 @@ class NormalizeLineBreaksSerializer : StdSerializer<String>(String::class.java)
 /**
  * Create an [Issue] and log the message. The log level is aligned with the [severity].
  */
-inline fun <reified T : Any> T.createAndLogIssue(source: String, message: String, severity: Severity? = null): Issue {
-    val issue = severity?.let { Issue(source = source, message = message, severity = it) }
-        ?: Issue(source = source, message = message)
+inline fun <reified T : Any> T.createAndLogIssue(
```

Comment:
Commit message nit: The comma before "which" reads wrong to me, it sounds like you'd be referring to another bug-issue.

Reply:
Removed the comma.